ROOTFS/etc/waggle/sanity/fatal/core_sdcard.test

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+pf="Recovery SDCard test"
+echo "$pf: Begin"
+
+# run fsck on our sd card
+if [ "$(fsck -n /dev/mmcblk1p1 | grep clean)" = "" ];
+then
+        echo "$pf: /dev/mmcblk1p1 failed fsck FAIL"
+        exit 1
+fi
+
+if [ "$(fsck -n  /dev/mmcblk1p2 | grep clean)" = "" ];
+then
+        echo "$pf: /dev/mmcblk1p2 failed fsck FAIL"
+        exit 2
+fi
+
+mkdir /tmp/core_sdcard_test
+
+mount /dev/mmcblk1p1  /tmp/core_sdcard_test/   
+
+if [ "$(cat /tmp/core_sdcard_test/etc/overlayroot.conf | grep "overlayroot=\"tmpfs")" = ""  ];
+then
+        echo "$pf: sdcard tmpfs not properly set up FAIL"
+        exit 3
+fi
+
+if [ "$?" != "0" ];
```

Comment:
Not a review comment, just wondering about how this test tells that you can't run chroot on the SD card?

Reply:
i agree with Omar.  i think you missed the command execution.  I think also you can do something like

`chroot /tmp/core_sdcard_test/ ls` to execute `ls` in the `chroot` environment.  if that command succeeds i think we are confident the rootfs is stable on the sdcard